crates/shared-crypto/src/intent.rs

```diff
@@ -35,6 +35,7 @@ impl TryFrom<u8> for IntentVersion {
 pub enum AppId {
     Sui = 0,
     Narwhal = 1,
+    Consensus = 2,
```

Comment:
CC @joyqvq @benr-ml to double check that those changes are ok

Reply:
the usage looks correct to me, signing commits to app_id + intent_scope + block_msg itself 👍

Reply:
I slightly prefer to have here a concrete protocol name (e.g., Bullshark) and leave Intent below to be more generic (e.g., ConsensusBlock).

Reply:
Btw why is AppId useful, when only a single IntentScope enum is used?

Reply:
> I slightly prefer to have here a concrete protocol name (e.g., Bullshark) and leave Intent below to be more generic (e.g., ConsensusBlock).

Not an issue with that per se, but we are now following a naming approach that is not tied to the protocol it self (mysticeti) - you can see that all the development is happening under the `Consensus` crate. So I would prefer following that convention here as well.